vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/private_cdn/get_publisher/get_publisher.cc

```diff
@@ -0,0 +1,268 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include "bat/ledger/internal/endpoint/private_cdn/get_publisher/get_publisher.h"
+
+#include <utility>
+
+#include "base/strings/string_util.h"
+#include "base/strings/stringprintf.h"
+#include "bat/ledger/internal/common/brotli_helpers.h"
+#include "bat/ledger/internal/common/time_util.h"
+#include "bat/ledger/internal/endpoint/private_cdn/private_cdn_util.h"
+#include "bat/ledger/internal/ledger_impl.h"
+#include "bat/ledger/internal/publisher/protos/channel_response.pb.h"
+#include "brave/components/brave_private_cdn/private_cdn_helper.h"
+#include "net/http/http_status_code.h"
+
+using std::placeholders::_1;
+
+// GET /publishers/prefixes/{prefix}
+//
+// Success code:
+// HTTP_OK (200)
+//
+// Error codes:
+// HTTP_NOT_FOUND (404)
+//
+// Response body:
+// See https://github.com/brave/brave-core/blob/master/vendor/bat-native-ledger/src/bat/ledger/internal/publisher/protos/channel_response.proto
```

Comment:
Starting to think these documentation blocks may make more sense in the header files, but totally understand if we don't want to do that right now.

Reply:
yeah I was wondering the same, please create and issue and let's move it to h

Reply:
Created https://github.com/brave/brave-browser/issues/11334.